cloud/deploys/dev_azure/civiform_config.example.sh

```diff
@@ -0,0 +1,34 @@
+export CIVIFORM_CLOUD_PROVIDER="azure"
+export CIVIFORM_MODE="dev"
+export CIVIC_ENTITY_SHORT_NAME="Sooschester"
+export CIVIC_ENTITY_FULL_NAME="City of Sooschester"
+export CIVIC_ENTITY_SUPPORT_EMAIL_ADDRESS="CHANGE ME"
+export CIVIC_ENTITY_LOGO_WITH_NAME_URL="https://www.cityofrochester.gov/images/header-logo.png"
+export CIVIC_ENTITY_SMALL_LOGO_URL="https://www.cityofrochester.gov/assets/0/117/8589934986/c6791498-270e-40b1-9b81-aca8a510c1b5.jpg"
+export EMAIL_SENDER="ses"
+export ADMIN_AUTH_PROVIDER="activeDirectory"
+export ADMIN_AUTH_FORMAT="OIDC"
+export RESIDENT_AUTH_PROVIDER="loginRadius"
+export APPLICATION_NAME="CHANGE ME"
+export DOCKER_REPOSITORY_NAME="civiform"
+export DOCKER_USERNAME="civiform"
+export IMAGE_TAG="latest"
+export SENDER_EMAIL_ADDRESS="CHANGE ME"
+export STAGING_HOSTNAME=""
+export STAGING_PROGRAM_ADMIN_NOTIFICATION_MAILING_LIST="CHANGE ME"
+export STAGING_TI_NOTIFICATION_MAILING_LIST="CHANGE ME"
+export STAGING_APPLICANT_NOTIFICATION_MAILING_LIST="CHANGE ME"
+export TERRAFORM_TEMPLATE_DIR="cloud/azure/templates/azure_saml_ses"
+export AZURE_LOCATION="eastus"
+export AZURE_LOG_STORAGE_ACCOUNT_NAME="CHANGE ME"
+export AZURE_SUBSCRIPTION="ec814e56-5241-4d0b-9cd2-02d608c99732"
+export AZURE_RESOURCE_GROUP="CHANGE ME"
+export AWS_USERNAME="CHANGE ME"
+export SAML_KEYSTORE_ACCOUNT_NAME="CHANGE ME"
+export CIVIFORM_APPLICANT_AUTH_PROTOCOL="saml"
+export CUSTOM_HOSTNAME=""
+export KEY_VAULT_NAME="civiform"
+export ADFS_ADMIN_GROUP="admins"
+export LOGIN_RADIUS_API_KEY="1b922870-0719-4970-94e3-cba7b62c5844"
```

Comment:
Is this a real API key?  We should probably put a fake in here, and guidance on how to generate your own

Reply:
this api key is public and signed in our requests